### PR TITLE
Create snippet for PLEG

### PR DESCRIPTION
**Pull request by thecrudge**
Addresses PLEG issue #13555

### CODE REVIEW

File: content/en/docs/concepts/_index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The Concepts section helps you learn about the parts of the Kubernetes system an
...
 
-Once you've set your desired state, the *Kubernetes Control Plane* works to make the cluster's current state match the desired state. To do so, Kubernetes performs a variety of tasks automatically--such as starting or restarting containers, scaling the number of replicas of a given application, and more. The Kubernetes Control Plane consists of a collection of processes running on your cluster: 
+Once you've set your desired state, the *Kubernetes Control Plane* using the Pod Lifecycle Event Generator (PLEG) works to make the cluster's current state matches the desired state. To do so, Kubernetes performs a variety of tasks automatically--such as starting or restarting containers, scaling the number of replicas of a given application, and more. The Kubernetes Control Plane consists of a collection of processes running on your cluster:
```
Comment (kbhawkey) [inline content/en/docs/concepts/_index.md]: Hello @thecrudge. Thanks for the PR.
- Check the commit message, see the note about referencing issues in commit messages:
https://kubernetes.io/docs/contribute/intermediate/#clone-the-repository
- I am having trouble parsing the changes.

Comment (thecrudge) [reply content/en/docs/concepts/_index.md]: Try now, I've amended my commit message

**Comment by DanyC97** [comment]
thanks for taking on this fix @thecrudge !
/lgtm

**Pull request merged**